tests/indicators/a-d/Adx/Adx.Catalog.Tests.cs

```diff
@@ -58,7 +58,7 @@ public void AdxBufferListing()
 
         // Assert
         listing.Should().NotBeNull();
-        listing.Name.Should().Be("Average Directional Index (ADX)");
+        listing.Name.Should().Be("Average Directional Index (ADX) (Buffer)");
```

Comment:
In sources, don’t add these suffix markers, we already have this info in the Style property. Many places.